### PR TITLE
[WIP]@orta=>moving all new settings work toward swift

### PR DESCRIPTION
**Pull request by sarahscott**
- [x] rewrote navigation controller in swift
- [x] added placeholder detail views to storyboard
- [ ] tests

### CODE REVIEW

File: Classes/Controllers/Settings/ARLabSettingsNavigationController.swift [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class ARLabSettingsNavigationController:UINavigationController {
+
+    func exitSettingsPanel(){
+        precondition(self.parentViewController is ARLabSettingsViewController, "Parent view controller must be an ARLabSettingsViewController")
```
Comment (orta) [inline Classes/Controllers/Settings/ARLabSettingsNavigationController.swift]: :+1: - I didn't know about this

**Comment by orta** [comment]
It's a bit weird that `Pods/Artsy+UILabels/Pod/Assets/Chevron_Gray@2x.png` got included

**Comment by sarahscott** [comment]
Getting this error when I try to test:

Include of non-modular header inside framework module 'Nimble.Nimble'

I tried this but to no avail: http://stackoverflow.com/questions/24103169/swift-compiler-error-non-modular-header-inside-framework-module

**Comment by orta** [comment]
@ashfurrow re the error ^

**Comment by ashfurrow** [comment]
Ugh, _this_ error. Keep running into it, and I don't have a clear solution to fix things. I just kind of play with it until it works. I'll check out this branch and see if I can get it to work.

**Comment by ashfurrow** [comment]
Tests compile fine for me – did a clone, checkout, `make artsy`, and `pod install`.

**Comment by ashfurrow** [comment]
(Though, the tests fail – not sure I can download the 8.4 simulator on hotel wifi.)

**Comment by sarahscott** [comment]
@ashfurrow I'll try a fresh clone. FWIW i'm using Xcode 7.1 - I might try it out on 7. Thanks for taking a look!

**Comment by ashfurrow** [comment]
Good idea! I was on 7.0

**Pull request closed without merging**